script.js

```diff
@@ -1,68 +1,87 @@
-let $inputApellidoDelUsuario = document.getElementById("inputApellidoDelUsuario");
-let $inputNombreDelUsuario = document.getElementById("inputNombreDelUsuario");
-let $inputPrecioDelUsuario = document.getElementById("inputPrecioDelUsuario");
-let $inputImpuestoDelUsuario = document.getElementById("inputImpuestoDelUsuario");
-let $textPrecioConImpuesto = document.getElementById("valorUsuarioConImpuesto");
-let $buttonEnviarValor = document.getElementById("buttonEnviarValor");
-let $inputDescuentoSobrePrecioFinal = document.getElementById("inputDescuentoSobrePrecioFinal");
-let $buttonAplicarDescuento = document.getElementById("buttonAplicarDescuento");
-let $textValorFinalConDescuento = document.getElementById("valorFinalConDescuento");
-let $textDatosDelUsuario = document.getElementById("datosDelUsuario");
-let $textNumerosOrdenados = document.getElementById("textNumerosOrdenados")
-
-const calcularImpuesto = (valorImpuesto) => 1 + valorImpuesto / 100;
-
-const calcularPrecioFinal = (valorUsuario, valorIvaUsuario) =>
-  valorUsuario * valorIvaUsuario;
-
-const calcularCantidadDescuento = (precioConImpuesto, descuento) =>
-  precioConImpuesto * (descuento / 100);
-
-const precioFinalConDescuento = (precioConImpuesto, descuentoSobrePrecio) =>
-  precioConImpuesto - descuentoSobrePrecio;
-
-const crearObjeto = (nombre, apellido) => {
+let $formImpuestos = document.getElementById("formImpuestos");
+let $inputCantidadDinero = document.getElementById("precio");
+let $inputCantidadImpuesto = document.getElementById("impuesto");
+let $textErrorValorImpuesto = document.getElementById("errorValorImpuesto");
+let $inputCantidadDescuento = document.getElementById("descuento");
+let $buttonEnviarDatos = document.getElementById("enviarDatos");
+let $rowMostarDatos = document.getElementById("mostarDatosEnTabla");
+
+const calcularImpuestos = (valorImpuesto) => 1 + valorImpuesto / 100;
+
+const calcularPrecioConImpuesto = (dineroIngresado, impuestosIngresado) =>
+  dineroIngresado * impuestosIngresado;
+
+const calcularDescuentoSobreImpuesto = (dineroConImpuesto, descuento) =>
+  dineroConImpuesto * (descuento / 100);
+
+const precioFinal = (dineroConImpuesto, descuentoSobrePrecio) =>
+  dineroConImpuesto - descuentoSobrePrecio;
+
+const crearPrecios = (dinero, impuesto, descuento) => {
   return {
-    nombre,
-    apellido,
-    obtenerDatos: () => {
-      return `${nombre} ${apellido}`;
-    },
+    dinero,
+    impuesto,
+    descuento,
   };
 };
 
-let impuestos = []
+let valoresFinal = [];
+
+const init = () => {
+  valoresFinal = JSON.parse(localStorage.getItem("valoresFinal")) || [];
+};
+
+init();
 
 let precioConImpuesto;
 let precioConDescuento;
-const handleClickEnviarValor = (e) => {
+
+const pintarRow = (valoresFinal) => {
```

Comment:
Aca veo que se llama pintarRow, pero estas pintando varias, deberiamos renombrarla

```suggestion
const pintarRows = (valoresFinal) => {
```